doxygen/pack.dxy

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "Open-CMSIS-Pack"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "Version 1.7.37"
+PROJECT_NUMBER         = "Version 1.7.38"
```

Comment:
Do we need to  update the schema version to match the project number? Or should we rather leave the project number untouched?

Reply:
Happy to revert.